servicetalk-http-api/src/test/java/io/servicetalk/http/api/HeaderUtilsTest.java

```diff
@@ -39,6 +42,17 @@
 import static org.junit.Assert.assertTrue;
 
 public class HeaderUtilsTest {
+    @Test
+    public void DEFAULT_DEBUG_HEADER_FILTER() {
+        assertEquals(APPLICATION_JSON, HeaderUtils.DEFAULT_DEBUG_HEADER_FILTER.apply(CONTENT_TYPE, APPLICATION_JSON));
+
+        assertEquals("3495", HeaderUtils.DEFAULT_DEBUG_HEADER_FILTER.apply(CONTENT_LENGTH, "3495"));
+
+        assertEquals(GZIP, HeaderUtils.DEFAULT_DEBUG_HEADER_FILTER.apply(TRANSFER_ENCODING, GZIP));
```

Comment:
Consider using `CHUNKED` here instead of `GZIP` as a more relevant for this use-case.